README.md

```diff
@@ -403,8 +405,12 @@ Error: No such file or directory @ rb_sysopen - C:/ProgramData/PuppetLabs/puppet
 Error: Try 'puppet help module install' for usage
   ~~~
 
-For Puppet 4.2.2+ (and 3.8.2) we've decreased the possibility of the issue occurring based on the fixes in [PUP-4854](https://tickets.puppetlabs.com/browse/PUP-4854). A complete fix will become available in a future version of Puppet that incorporates [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866).
-If you are affected by this issue, you can work around this by downloading the `.tar.gz` from the Forge and use `puppet module install` using the file rather than directly from the Forge.
+  For Puppet 4.2.2+ (and 3.8.2) we've decreased the possibility of the issue occurring based on the fixes in [PUP-4854](https://tickets.puppetlabs.com/browse/PUP-4854). A complete fix is plannd in a future version of Puppet that incorporates [PUP-4866](https://tickets.puppetlabs.com/browse/PUP-4866).
+  
+  If you are affected by this issue, you can work around this by;
+  - Use the `--module_working_dir` parameter to set a different temporary directory which has a smaller length, for example;
+    `puppet module install puppetlabs-dsc --module_working_dir C:\Windows\Temp`
+  - Downloading the `.tar.gz` from the [Forge](https://forge.puppetlabs.com/puppetlabs/dsc) and use `puppet module install` using the file rather than directly from the Forge.
```

Comment:
Sorry, should've made this two different comments. In addition to adding the link, let's edit this  bullet so it reads:

```
Download the `.tar.gz` from the [Forge](https://forge.puppetlabs.com/puppetlabs/dsc) and use `puppet module install` using the downloaded file rather than directly installing from the Forge.
```